sdk/containerregistry/container-registry/swagger/containerregistry.json

```
@@ -1700,7 +1689,8 @@
           "type": "string"
         }
       },
-      "x-accessibility": "internal"
+      "x-accessibility": "internal",
+      "x-ms-client-name": "AcrRefreshToken"
     },
     "AccessToken": {
```

Similarly, could we add this to get `RegistryArtifactOrderBy` as an auto-generated extensible enum?  This may not be the ideal place to put it, but was the best place I could find on the PR diff.

```suggestion
    "ManifestOrderBy": {
      "type": "string",
      "enum": [
        "time_desc",
        "time_asc"
      ],
      "x-ms-enum": {
        "name": "RegistryArtifactOrderBy",
        "modelAsString": true,
        "values": [
          {
            "value": "time_desc",
            "name": "LastUpdatedOnDescending",
            "description": "Order registry artifacts by LastUpdatedOn field, from most recently updated to least recently updated."
          },
          {
            "value": "time_asc",
            "name": "LastUpdatedOnAscending",
            "description": "Order  registry artifacts by LastUpdatedOn field, from least recently updated to most recently updated."
          }
        ]
      },
      "default": "none",
      "description": "Sort options for ordering registry artifacts in a collection."
    },
    "AccessToken": {
```